client-app/shared/static-content/components/slider.vue

```diff
@@ -28,16 +28,26 @@
 </template>
 
 <script setup lang="ts">
+import { useBreakpoints } from "@vueuse/core/index";
 import { Navigation } from "swiper";
 import { Swiper, SwiperSlide } from "swiper/vue"; // eslint-disable-line import/no-unresolved
 import { computed, getCurrentInstance } from "vue";
+import { BREAKPOINTS } from "@/core/constants";
 
 const props = defineProps<{
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   model: Record<string, any>;
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   settings: Record<string, any>;
 }>();
+const breakpoints = useBreakpoints(BREAKPOINTS);
+
+/* CLS fix */
+const imageHeights = { xs: 250, sm: 250, md: 400, lg: 400, xl: 450, "2xl": 450 };
+const imageHeight = computed(() => {
+  const currentBreakpoint = (breakpoints.current().value.at(-1) || "xs") as keyof typeof imageHeights;
+  return imageHeights[currentBreakpoint] + "px";
+});
```

Comment:
Mobile is < xs and `breakpoints.current` will return array of all breakpoints, instead of the one which meet current viewport size, so use the same approach with `if` as I used for lazy loading images